### PR TITLE
Manage default value in filters

### PR DESCRIPTION
**Pull request by jc7447**
Not sure if this is the good place to contribute, let me know :)

The goal is to manage defaut value  like ${toto:tutu} where tutu is used when no value for toto is found.
It is the same behavior as Spring's [PropertyPlaceholderConfigurer](https://www.mkyong.com/spring/spring-propertyplaceholderconfigurer-example/)

### CODE REVIEW

**Comment by eolivelli** [comment]
Sure, can you please create an issue here

https://issues.apache.org/jira/browse/MSHARED

Just follow the guide

https://maven.apache.org/guides/development/guide-helping.html

Thanks

File: src/test/java/org/apache/maven/shared/filtering/MultiDelimiterInterpolatorFilterReaderLineEndingTest.java [hunk 1]
```diff
@@ -121,4 +123,25 @@ public void testAtDollarExpression()
 
         assertEquals( "  url=\"jdbc:oracle:thin:@DB_SERVER:DB_PORT:DB_NAME\"", IOUtil.toString( reader ) );
     }
+    
+    @Test
+    public void testDefaultValue()
```
Comment (eolivelli) [inline src/test/java/org/apache/maven/shared/filtering/MultiDelimiterInterpolatorFilterReaderLineEndingTest.java]: What about adding tests to cover syntax error cases? Or empty string as default...
